llvm/include/llvm/Support/PGOOptions.h

```diff
@@ -27,10 +27,12 @@ class FileSystem;
 struct PGOOptions {
   enum PGOAction { NoAction, IRInstr, IRUse, SampleUse };
   enum CSPGOAction { NoCSAction, CSIRInstr, CSIRUse };
+  enum class ColdFuncAttr { None, OptSize, MinSize, OptNone };
```

Comment:
Even though the optimization directive is implemented as a function attribute, it might be better to express the real intention is to decide 'optimization strategy', so perhaps ColdFuncOpt as name or ColdFuncOptAttr? ColdFuncAttr feels too broad.